modules/standard/SysBasic.chpl

```diff
@@ -98,227 +175,757 @@ inline proc c_calloc(type eltType, size: integral) {
+
+/* Operation already in progress. An operation was attempted on a non-blocking
+   object that already had an operation in progress. (POSIX.1)
+ */
 extern const EALREADY:err_t;
+
+/* Invalid exchange (linux only) */
 extern const EBADE:err_t;
+
+/* Bad file descriptor. A file descriptor argument was out of range, referred
+   to no open file, or a read (write) request was made to a file that was only
+   open for writing (reading). (POSIX.1)
+  */
 extern const EBADF:err_t;
+
+/* File descriptor in bad state (linux only) */
 extern const EBADFD:err_t;
+
+/* Bad message. A corrupted message was detected. (POSIX.1) */
 extern const EBADMSG:err_t;
+
+/* Invalid request descriptor (linux only) */
 extern const EBADR:err_t;
+
+/* Invalid request code (linux only) */
 extern const EBADRQC:err_t;
+
+/* Invalid slot (linux only) */
 extern const EBADSLT:err_t;
+
+/* Device or resource busy. An attempt to use a system resource which was in
+   use at the time in a manner which would have conflicted with the request.
+   (POSIX.1)
+ */
 extern const EBUSY:err_t;
+
+/* Operation canceled. The scheduled operation was canceled. (POSIX.1) */
 extern const ECANCELED:err_t;
+
+/* No child processes. A wait or waitpid system call was executed by a
+   process that had no existing or unwaited-for child processes. (POSIX.1)
+ */
 extern const ECHILD:err_t;
+
+/* Channel number out of range (linux only) */
 extern const ECHRNG:err_t;
+
+/* Communication error on send (linux only) */
 extern const ECOMM:err_t;
+
+/* Software caused connection abort. A connection abort was caused internal
+   to your host machine. (POSIX.1)
+ */
 extern const ECONNABORTED:err_t;
+
+/* Connection refused. No connection could be made because the target machine
+   actively refused it. This usually results from trying to connect to a
+   service that is inactive on the foreign host. (POSIX.1)
+ */
 extern const ECONNREFUSED:err_t;
+
+/* Connection reset by peer. A connection was forcibly closed by a peer. This
+   normally results from a loss of the connection on the remote socket due to a
+   timeout or a reboot. (POSIX.1) 
+ */
 extern const ECONNRESET:err_t;
+
+/* Resource deadlock avoided. An attempt was made to lock a system resource
+   that would have resulted in a deadlock situation. (POSIX.1)
+ */
 extern const EDEADLK:err_t;
+
+/* Destination address required. A required address was omitted from an
+   operation on a socket. (POSIX.1)
+ */
 extern const EDESTADDRREQ:err_t;
+
+/* Numerical argument out of domain. A numerical input argument was outside
+   the defined domain of the mathematical function. (POSIX.1, C99)
+ */
 extern const EDOM:err_t;
+
+/* Disc quota exceeded. A write system call to an ordinary file, the creation
+   of a directory or symbolic link, or the creation of a directory entry failed
+   because the user's quota of disk blocks was exhausted, or the allocation of
+   an inode for a newly cre- ated file failed because the user's quota of inodes
```

Comment:
There's an odd "- " in the middle of "created", I'm guessing that wasn't intentional.